handbook/technical-overview/postholf/postholf-03-interface-skjalaveita.md

```diff
@@ -1,16 +1,16 @@
 # Skjalaveita API
 
-Service that document providers need to implement. All of the document providers need to implement the same interface. The backend system in island.is will call this service to retrieve document from document provider when a user wants to view the document.
+Service that document providers need to implement. All of the document providers need to implement the same interface. The backend system in Island.is will call this service to retrieve documents from the document provider when a user wants to view the document.
 
-Https communication is required. The backend system will identify itself with JWT in the Authorization header using the Bearer schema. The service MUST validate the signature, issuer, expiry dates, audience and scope
+HTTPS communication is required. The backend system will identify itself with JWT in the Authorization header using the Bearer schema. The service MUST validate the signature, issuer, expiry dates, audience and scope
 
 ## Document
 
-The operation returns a owner's document. The service should only return a document if the identifier (SkjalId) and owner kennitala matches in the document provider systems.
+The operation returns an owners document. The service should only return a document if the identifier (SkjalId) and owner kennitala matches in the document provider systems.
```

Comment:
"owners" is plural
"owner's" is to [mark possession](https://www.lynchburg.edu/academics/writing-center/wilmer-writing-center-online-writing-lab/grammar/a-quick-guide-to-punctuation/). Eg "John's document", "owner's document".